foundation-ui/mount/Mount.js

```diff
@@ -102,7 +102,7 @@ Mount.propTypes = {
   limit: PropTypes.number,
   children: PropTypes.element,
   type: PropTypes.string.isRequired,
-  wrapper: PropTypes.node
+  wrapper: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
```

Comment:
Yes, this is what React does itself: https://github.com/facebook/react/blob/c78464f8ea9a5b00ec80252d20a71a1482210e57/src/isomorphic/classic/element/ReactElementValidator.js#L186